conanfile.py

```diff
@@ -11,17 +11,25 @@ class DiceHashConan(ConanFile):
     author = "DICE Group <info@dice-research.org>"
     homepage = "https://github.com/dice-group/dice-hash"
     url = homepage
-    topics = ("hash", "wyhash", "xxh3", "robin-hood-hash", "C++", "C++20")
-    settings = "build_type", "compiler", "os", "arch"
-    generators = ("CMakeDeps", "CMakeToolchain")
+    topics = ("hash", "wyhash", "xxh3", "robin-hood-hash", "Blake2b", "Blake2Xb", "LtHash", "C++", "C++20")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"with_test_deps": [True, False], "with_sodium": [True, False]}
+    default_options = {"with_test_deps": False, "with_sodium": False}
     exports = "LICENSE"
-    exports_sources = "include/*", "CMakeLists.txt", "cmake/*", "LICENSE"
-    options = {
-        "with_test_deps": [True, False],
-    }
-    default_options = {
-        "with_test_deps": False,
-    }
+    exports_sources = "src/*", "CMakeLists.txt", "cmake/*", "LICENSE"
+
+    generators = ("CMakeDeps", "CMakeToolchain")
+
+    def requirements(self):
+        if self.options.with_sodium:
+            self.requires("libsodium/cci.20220430")
+        self.requires("highway/1.2.0")
```

Comment:
I think it would make sense to rename the option `with_blake` or something along those lines and move the `highway` require into the branch